code/modules/mob/living/simple_animal/guardian/guardian.dm

```diff
@@ -598,21 +603,123 @@
 /mob/living/simple_animal/hostile/guardian/protector
 	melee_damage_lower = 15
 	melee_damage_upper = 15
-	damage_coeff = list(BRUTE = 0.3, BURN = 0.3, TOX = 0.3, CLONE = 0.3, STAMINA = 0, OXY = 0.3)
-	playstyle_string = "As a protector type you have very high damage resistance, a decent attack, and cause your summoner to leash to you instead of you leashing to them."
-	magic_fluff_string = "..And draw the Guardian, a stalwart protector that never leaves the side of its charge."
-	tech_fluff_string = "Boot sequence complete. Protector modules loaded. Holoparasite swarm online."
+	range = 15 //worse for it due to how it leashes
+	damage_coeff = list(BRUTE = 0.4, BURN = 0.4, TOX = 0.4, CLONE = 0.4, STAMINA = 0, OXY = 0.4)
+	playstyle_string = "<span class='holoparasite'>As a <b>protector</b> type you cause your summoner to leash to you instead of you leashing to them and have two modes; Combat Mode, where you do and take medium damage, and Protection Mode, where you do and take almost no damage, but move slightly slower.</span>"
+	magic_fluff_string = "<span class='holoparasite'>..And draw the Guardian, a stalwart protector that never leaves the side of its charge.</span>"
+	tech_fluff_string = "<span class='holoparasite'>Boot sequence complete. Protector modules loaded. Holoparasite swarm online.</span>"
+	var/toggle = FALSE
+
+/mob/living/simple_animal/hostile/guardian/protector/ex_act(severity)
+	if(severity == 1)
+		adjustBruteLoss(400) //if in protector mode, will do 20 damage and not actually necessarily kill the summoner
+	else
+		..()
+	if(toggle)
+		visible_message("<span class='danger'>The explosion glances off [src]'s energy shielding!</span>")
+
+/mob/living/simple_animal/hostile/guardian/protector/ToggleMode()
+	if(cooldown > world.time)
+		return 0
+	cooldown = world.time + 10
+	if(toggle)
+		overlays.Cut()
```

Comment:
Do you know if overlays is used elsewhere? Using cut() without specifying arguments will delete the entire list